crates/biome_js_analyze/tests/specs/nursery/useAriaPropsSupportedByRole/valid.jsx

```diff
@@ -56,9 +55,8 @@
 <link href="#" aria-live />
 <link href="#" aria-owns />
 <link href="#" aria-relevant />
-<img alt="" aria-checked />
 <img alt="foobar" aria-busy />
-<menu type="toolbar" aria-activedescendant />
```

Comment:
Same as [the previous case.](https://github.com/biomejs/biome/pull/4488/files#r1833385902)